WordPress/gutenberg
# Add a GlobalStylesScreenRootSlot slot in Global Styles root screen

## What?
Resolves: https://github.com/WordPress/gutenberg/issues/48068

Add a `GlobalStylesScreenRootSlot` slot in the Global Styles root screen.
## Why?

In order to allow integrators a way to add custom elements in the Global Styles Screen Root, a new slot is added at the top of the section. 
## How?

It creates a new slot that's called at the top of the GlobalStylesScreenRoot component.
## Testing Instructions




It doesn't change the current behavior. 
Open the Global Styles in site editor and make sure nothing changes.
### Testing Instructions for Keyboard
